### ISSUE TITLE
Remove a User's ability to "cancel" their RSVP

### ISSUE THREAD

**Issue by dreazee** [SDRefugeeTutoring/sdrt-custom-functions#50]
@mathetos @JasonTheAdams Please remove the option for a user to cancel their RSVP. It actually does not work anyway and it just confuses the User. 
We are trying to promote more conversation between a Volunteer (User) and SDRT staff regarding their once-a-week commitment, and the ability for a user to cancel their own RSVP gets in the way of this.
I literally fielded maybe 6 emails today from people saying they tried to cancel on the website but they couldn't so they emailed instead. We don't want people cancelling on the website, we want people to email. And we had 1 email from someone who received a confirmation email for tutoring but thought they had cancelled their RSVP online. VERY CONFUSING.

**Comment by mathetos** [comment]
@JasonTheAdams To clarify, after a user has RSVP'd, they'll see the below message:
![image](https://user-images.githubusercontent.com/930724/115815564-08aa1a00-a3ac-11eb-91c0-5565b2bcc6f6.png)

The "Cancel Here" is what is not working currently. This is output here:
https://github.com/SDRefugeeTutoring/sdrt-custom-functions/blob/c4519cdc22b7f59bed74ac22dc7ccd1ae2001995/plugin_customizations/tribe_events/tribe_events_customizations.php#L231-L236

Let's just change the wording to:
> Need to cancel? Please email our Volunteer Coordinator at info@sdrefugeetutoring.com

**Comment by JasonTheAdams** [comment]
Hi @mathetos and @dreazee!

Per Matt's direction, this is all set:
![image](https://user-images.githubusercontent.com/2024145/115976182-21d3d780-a520-11eb-8a90-02009386ed52.png)

While I was in there I did find why the cancel button wasn't working. In fact, all users, regardless of whether they were marked as attending, would see the message intended for users marked to attend. This bug has been resolved. It still sounded like the new intent is to have users email instead of doing it themselves, so that's how it works now.

Since this was such a minor change I didn't bother to write up a PR and just pushed this straight to production. 🎉
